GameDefinition/PLAYING.md

```diff
@@ -527,6 +539,99 @@ influence the speed of HP attrition and if it outweighs the benefits
 of exploration.
 
 
+Managing inventory and crafting
+-------------------------------
+
+All items belonging to your team are stored either in the character's local
+inventory (called the character's outfit) or in the shared inventory that
```

Comment:
I'd stick names with mutually disjoint sets of words, that is, any abbreviations of these full forms: "personal equipment outfit", "shared inventory stash". Notice that this matches the keys used for management. If you say "local inventory" the character may try to use key I to view it.

Reply:
True. I'll check it again.